r_code/group_A.R

```diff
@@ -0,0 +1,153 @@
+#############################
+## Load required libraries ##
+#############################
+
+library(DESeq2)
+library(ggplot2)
+library(data.table)
+
+#################################
+## Defining output directories ##
+#################################
+
+## Main output directory:
+OUTPUT_DIR <- file.path("results")
+dir.create(OUTPUT_DIR, recursive = TRUE, showWarnings = FALSE)
+
+## Generated data:
+DATA_DIR <- file.path(OUTPUT_DIR, "data")
+dir.create(DATA_DIR, recursive = TRUE, showWarnings = FALSE)
+
+## Produced results:
+RESULTS_DIR <- file.path(OUTPUT_DIR, "results")
+dir.create(RESULTS_DIR, recursive = TRUE, showWarnings = FALSE)
+
+#########################
+## Simulating the data ##
+#########################
+
+COUNT_MAT_PATH <- file.path(DATA_DIR, "count_matrix.RData")
+METADATA_PATH  <- file.path(DATA_DIR, "metadata.tsv")
+
+## DATA 1
+## Create example RNA-seq count matrix and export it
+
+set.seed(567)
+num_genes <- 1000
+num_samples <- 10
+
+count_matrix <- matrix(data = round(runif(num_genes * num_samples, min = 0, max = 100)), nrow = num_genes)
```

Comment:
since you do missing data imputation, maybe here you can have a line that removes genes at random?
another thing: I would not add this code into this script, but provide only the data. We can keep this file elsewhere for reference